modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/pages/diff-view-base.hbs

```diff
@@ -47,6 +47,9 @@
 
 {{include listassets}}
 
+<!--  CSRF Protection -->
+<script src='{{url ""}}/csrf.js' type="text/javascript"></script>
+
```

Comment:
AFAIU is it safe to add comment related to security vulnerabilities? file name(csrf.js) is obvious too.